.changeset/shy-schools-end.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": minor
+---
+
+Formatter option `objectWrap` is now also supported in JSON files
```

Comment:
Since this feature was created but not yet released to the users, we can update `gentle-spies-drum.md`, and explain that it's also supported for JSON files

Reply:
@ematipico Thank you for the suggestion, updated the previous changeset in [065936f](https://github.com/biomejs/biome/pull/5082/commits/065936f89419b59dd875b2ff1955511de99702c8).